gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/EmergePlugin.kt

```diff
@@ -326,6 +333,38 @@ class EmergePlugin : Plugin<Project> {
     }
   }
 
+  private fun configureAppProject(
```

Comment:
µ-nit: totally optional but can the naming reflect it only configures if the `fromMainSourceSet` is set